dynamometer-workload/src/test/java/com/linkedin/dynamometer/workloadgenerator/TestWorkloadGenerator.java

```diff
@@ -105,5 +111,12 @@ private void testAuditWorkload() throws Exception {
     assertTrue(dfs.getFileStatus(new Path("/tmp/test1")).isFile());
     assertTrue(dfs.getFileStatus(new Path("/tmp/testDirRenamed")).isDirectory());
     assertFalse(dfs.exists(new Path("/denied")));
+
+    assertTrue(dfs.exists(new Path(auditOutputPath)));
+    FSDataInputStream auditOutput = dfs.open(new Path(auditOutputPath + "/part-r-00000"));
+    byte[] buf = new byte[auditOutput.available()];
+    assertTrue(auditOutput.read(buf) > 0);
+    System.out.println(new String(buf));
+    assertTrue(new String(buf).matches(".*hdfs,WRITE\\t[0-9]+\\n.*"));
```

Comment:
Since we actually know how many commands to expect for every user, I would prefer to assert on exact output. Then we ensure the counting is working properly.

Reply:
The reducer output is the summed latencies, so it's not exactly consistent across runs.

Reply:
Oh, right. I was thinking it was a count. Okay, SGTM.